hnn_core/gui/gui.py

```diff
@@ -518,6 +796,42 @@ def run_notebook_cells(self):
         """
         return js_string
 
+    # below are a series of methods that are used to manipulate the GUI
+    def _simulate_upload_file(self, file_url):
```

Comment:
use this also in `test_gui.py` ?

Reply:
did you address this @chenghuzi ?

Reply:
here is the part of the test I am imagining will become simpler: https://github.com/jonescompneurolab/hnn-core/blob/1170bbc1e1c046128e4dfc8b3ae09ba1feb1d138/hnn_core/tests/test_gui.py#L33-L50

Reply:
It's now fixed in our latest commit